src/platform-implementation-js/dom-driver/gmail/views/gmail-thread-view.ts

```diff
@@ -908,99 +906,73 @@ class GmailThreadView {
     return false;
   }
 
-  _setupMessageViewStream() {
-    var openMessage = this._element.querySelector('.h7');
+  #setupMessageViewStream() {
+    var openMessage = this.#element.querySelector('.h7');
 
     if (!openMessage) {
       var self = this;
       setTimeout(function () {
-        if (self._element) {
-          self._setupMessageViewStream();
+        if (self.#element) {
+          self.#setupMessageViewStream();
         }
       }, 500);
       return;
     }
 
     var messageContainer: HTMLElement = openMessage.parentElement as any;
 
-    this._initializeExistingMessages(messageContainer);
+    this.#initializeExistingMessages(messageContainer);
 
-    this._observeNewMessages(messageContainer);
+    this.#observeNewMessages(messageContainer);
   }
 
-  _initializeExistingMessages(messageContainer: any) {
+  #initializeExistingMessages(messageContainer: any) {
     var self = this;
     var children = messageContainer.children;
     Array.prototype.forEach.call(children, function (childElement) {
-      self._createMessageView(childElement);
+      self.#createMessageView(childElement);
     });
   }
 
-  _observeNewMessages(messageContainer: any) {
-    this._newMessageMutationObserver = new MutationObserver(
-      this._handleNewMessageMutations.bind(this),
+  #observeNewMessages(messageContainer: any) {
+    this.#newMessageMutationObserver = new MutationObserver(
+      this.#handleNewMessageMutations.bind(this),
     );
 
-    this._newMessageMutationObserver.observe(messageContainer, {
+    this.#newMessageMutationObserver.observe(messageContainer, {
       childList: true,
     });
   }
 
-  _handleNewMessageMutations(mutations: MutationRecord[]) {
+  #handleNewMessageMutations(mutations: MutationRecord[]) {
     var self = this;
     mutations.forEach(function (mutation) {
       Array.prototype.forEach.call(mutation.addedNodes, function (addedNode) {
         if (!addedNode.classList.contains('inboxsdk__custom_message_view'))
-          self._createMessageView(addedNode);
+          self.#createMessageView(addedNode);
       });
     });
   }
 
-  _createMessageView(messageElement: HTMLElement) {
-    var messageView = new GmailMessageView(messageElement, this, this._driver);
+  #createMessageView(messageElement: HTMLElement) {
+    var messageView = new GmailMessageView(messageElement, this, this.#driver);
 
-    this._eventStream.plug(messageView.getEventStream());
+    this.#eventStream.plug(messageView.getEventStream());
 
-    this._messageViewDrivers.push(messageView);
+    this.#messageViewDrivers.push(messageView);
 
-    this._eventStream.emit({
+    this.#eventStream.emit({
       type: 'internal',
       eventName: 'messageCreated',
       view: messageView,
     });
   }
 
-  _setupWidthManager() {
-    let widthManager = this._widthManager;
```

Comment:
This method doesn't ever seem to be called. With true private used for the method name, `#` prefixed, the TS Server points out that it can be dropped.

This is the only instance of `WidthManager` which can also be dropped.